README.md

```diff
@@ -1,130 +1,20 @@
-# File system notifications for Go
+# WARNING
 
-[![GoDoc](https://godoc.org/github.com/fsnotify/fsnotify?status.svg)](https://godoc.org/github.com/fsnotify/fsnotify) [![Go Report Card](https://goreportcard.com/badge/github.com/fsnotify/fsnotify)](https://goreportcard.com/report/github.com/fsnotify/fsnotify)
+If you are reading this, you use `master` branch of this repository,
+which is wrong.
 
-fsnotify utilizes [golang.org/x/sys](https://godoc.org/golang.org/x/sys) rather than `syscall` from the standard library. Ensure you have the latest version installed by running:
+This branch
+ - should not be used;
+ - is not maintained;
+ - is not supported;
+ - will be removed soon.
```

Comment:
Thanks @kolyshkin 

Do we want to say "soon"? I'm not sure when it will be removed...

Reply:
I wrote "in 2024" initially, but when I looked at it from user's perspective, I realized that "in 2024" reads like "there's nothing to worry about, you can relax for now" which is not the message I want to send.

So, I'd rather say "soon", having in mind a very broad definition of the term. :)

I can remove it if you feel it doesn't work.

Reply:
That makes sense. Let's keep "soon"